### PR TITLE
Enhancements for yogi

### PR DESCRIPTION
**Pull request by NickGrantX**
Enhanced the sample generator to be able to cope with different types of case (unit vs establishment)

### CODE REVIEW

**Comment by NickGrantX** [comment]
[Enhance sample generator for Yogi](https://trello.com/c/xfL6kHmn/1110-enhance-sample-generator-for-yogi)

File: generate_sample_file.py [hunk 1]
```diff
@@ -141,38 +144,24 @@ def generate_sample_file(self, output_file_path: Path, treatment_code_quantities
...
                     if treatment_code["treatment_code"][:3] == "SPG":
-                        self._write_estab_case(writer, sequential_uprn, treatment_code, address_type='SPG')
+                        self._write_spg_or_ce_case(writer, sequential_uprn, treatment_code, address_type='SPG',
+                                                   address_level=treatment_code["address_level"])
                         continue
 
-                    # Randomly decide whether to create a CE case or not
-                    if self.random_1_in_11():
-                        self._write_estab_case(writer, sequential_uprn, treatment_code, address_type='CE')
+                    if treatment_code["treatment_code"][:2] == "CE":
```
Comment (ryangrundy7) [inline generate_sample_file.py:151]: Any idea if we're gonna get CE treatment codes soon so we can put it in the sample generator? At the moment it'll just generate HH and SPG

**Review by hbrace — APPROVED** [review]

File: generate_sample_file.py (hunk 1 above)
Comment (NickGrantX) [reply generate_sample_file.py:151]: no idea

Comment (NickGrantX) [reply generate_sample_file.py:151]: no idea

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**